Official-Free-Code-Camp-Chat-Rooms.md

```diff
@@ -29,6 +29,6 @@ The following are our official chat rooms.
 | [CurriculumDevelopment](https://gitter.im/freecodecamp/CurriculumDevelopment) | help us improve our open source curriculum |
 | [DataScience](https://gitter.im/freecodecamp/DataScience) | help us understand our gigs and gigs of public data |
 
-By joining these chat rooms, you accept our [Code of Conduct](https://github.com/FreeCodeCamp/freecodecamp/wiki/Code-of-Conduct).
+By joining these chat rooms, you accept our [Code of Conduct](Code-of-Conduct).
```

Comment:
We would have to check with the core team on this. They created both of them, so will they keep it in sync, should we keep it in sync or just use the one on the site?